nixos/modules/misc/ids.nix

```diff
@@ -339,6 +339,7 @@
       rss2email = 312;
       cockroachdb = 313;
       zoneminder = 314;
+      nsq = 315;
```

Comment:
I guess we can avoid allocating a user, by using the `DynamicUser` feature of systemd.

Reply:
And if `DynamicUser` doesn't work, just a normal user without a static uid will work too